Languages/en.ini

```diff
@@ -679,6 +680,7 @@ TextureColorDepth=Choose the color depth of textures.\n\n32-bit textures support
 SmoothLines=Toggle whether certain theme components are rendered with antialiased lines.
 FastNoteRendering=If enabled, the z buffer is not cleared after every note.  This causes 3D noteskins to collide, but substantially improves performance in gameplay.
 ShowStats=Display performance statistics in the upper right corner of the screen and show frame skips in the lower right.\n\nFPS = frames per second\nVPF = vertices per frame\n\nA high VPF count indicates that the theme is drawing complex things and requires better hardware (or better programming) to maintain a steady FPS.\n\nThis feature can be toggled at any time by pressing F3+6.
+GlobalOffsetSeconds=Set the audio offset. Negative values cause the audio to be played earlier (late arrows), positive values cause audio playback to be late (early arrows).

```

Comment:
Should mention the unit is in milliseconds.

Reply:
I updated the description. ("Set the audio offset in milliseconds.")